packages/next/build/babel/loader/types.d.ts

```diff
@@ -16,4 +16,5 @@ export interface NextBabelLoaderOptions {
   caller: any
   configFile: string | undefined
   cwd: string
+  pageExtensions: string[]
```

Comment:
```suggestion
  nextConfig: NextConfigComplete
```

Let's mirror the SWC loader options. You can import this from:

```ts
import type { NextConfigComplete } from '../../../server/config-shared'
```